tests/models/bloom/test_modeling_bloom.py

```diff
@@ -435,24 +435,26 @@ def test_simple_generation(self):
         )
 
         input_ids = tokenizer.encode(input_sentence, return_tensors="pt")
-        greedy_output = model.generate(input_ids.cuda(), max_length=50)
+        greedy_output = model.generate(input_ids.to(torch_device), max_length=50)
 
         self.assertEqual(tokenizer.decode(greedy_output[0], skip_special_tokens=True), EXPECTED_OUTPUT)
 
     @slow
     @require_torch_gpu
     def test_batch_generation(self):
         path_560m = "bigscience/bloom-560m"
-        model = BloomForCausalLM.from_pretrained(path_560m, use_cache=True, revision="gs555750").cuda()
+        model = BloomForCausalLM.from_pretrained(path_560m, use_cache=True, revision="gs555750").to(torch_device)
         model = model.eval()
         tokenizer = BloomTokenizerFast.from_pretrained(path_560m, padding_side="left")
 
         input_sentence = ["I enjoy walking with my cute dog", "I enjoy walking with my cute dog"]
 
         input_ids = tokenizer.batch_encode_plus(input_sentence, return_tensors="pt", padding=True)
-        greedy_output = model.generate(
-            input_ids["input_ids"].cuda(), attention_mask=input_ids["attention_mask"], max_length=50, do_sample=False
+        input_ids, attention_mask = (
+            input_ids["input_ids"].to(torch_device),
+            input_ids["attention_mask"],
         )
```

Comment:
```suggestion
        input_ids, attention_mask = input_ids["input_ids"].to(torch_device), input_ids["attention_mask"]
```
sorry for being noisy 😄

Reply:
This one is definitely a `make style` thing, I wrote it as a two liner and then it reformatted to this 😅

Reply:
then we can just do 
```suggestion
        input_ids = input_ids["input_ids"].to(torch_device)
        attention_mask =  input_ids["attention_mask"]
``` 
to fit in two lines!